### PR TITLE
`ls`: upgrade uutils-term-grid to 0.5

### PR DESCRIPTION
**Pull request by tertsdiepraam**
Draft PR for now, because we need to release a new version of term-grid first, but I wanted to show how the new API changes in https://github.com/uutils/uutils-term-grid/pull/27 are affecting `ls`.

### CODE REVIEW

**Comment by sylvestre** [comment]
@tertsdiepraam any plan here ? :)
thanks

File: src/uu/ls/Cargo.toml [hunk 1]
```diff
@@ -15,9 +15,9 @@ edition = "2021"
 path = "src/ls.rs"
 
 [dependencies]
+ansi-width = "0.1.0"
 clap = { workspace = true, features = ["env"] }
```
Comment (BenWiederhake) [inline src/uu/ls/Cargo.toml:19]: The fact that every entry except this one says `workspace = true` and doesn't provide a specific version number makes me wonder why it's the other way around here.

Comment (tertsdiepraam) [reply src/uu/ls/Cargo.toml:19]: Because I forgot 😅

File: src/uu/ls/Cargo.toml [hunk 2]
```diff
@@ -15,9 +15,9 @@ edition = "2021"
 path = "src/ls.rs"
 
 [dependencies]
+ansi-width = "0.1.0"
```
Comment (sylvestre) [inline src/uu/ls/Cargo.toml:18]:
```suggestion
ansi-width= { workspace = true }
```

**Comment by tertsdiepraam** [comment]
Ah I was just gonna do that. Thanks :)

**Pull request merged**